test/WebJobs.Script.Tests.Shared/TestHelpers.cs

```diff
@@ -112,6 +112,19 @@ public static void ClearHostLogs()
             }
 
             DirectoryInfo directory = GetFunctionLogFileDirectory(functionName);
+
+            if (!directory.Exists)
+            {
+                if (throwOnNoLogs)
+                {
+                    throw new InvalidOperationException($"{directory.FullName} does not exist.");
```

Comment:
It would probably be helpful to have a more descriptive message here (e.g. "Function log directory <dir> does not exist.")

Reply:
Yeah good point.